### PR TITLE
Update ab-testing.md

### PR DESCRIPTION
**Pull request by marialivia16**
As someone who hasn't contributed to frontend before, I added a few thing that would have been useful to know before starting to work on AB tests.

### CODE REVIEW

File: docs/ab-testing.md [hunk 1]
```diff
@@ -136,6 +136,12 @@ can follow our standard [contributing guidelines](https://github.com/guardian/fr
 
 You can stop and start the test using our [switchboard](https://frontend.gutools.co.uk/dev/switchboard).
 
+To see the test in action locally: `localhost:9000/[articleId]#ab-[testName]=[variantName]`
+
+To see the tests you are part of: open the Dev Console -> Resources -> Local Storage -> choose `http://localhost:9000` -> check the `gu.ab.participations` row.
```
Comment (NataliaLKB) [inline docs/ab-testing.md]: This is really useful regardless of whether you are on `localhost`, or the `theguardian.com`. Could we update this to clarify that this works on all stages (PROD, CODE and local) of the site?

File: docs/ab-testing.md [hunk 2]
```diff
@@ -136,6 +136,12 @@ can follow our standard [contributing guidelines](https://github.com/guardian/fr
...
+
+Tip: To make sure your code is compiled, before running `./sbt`, try `make compile-dev`. This might save you lots of time debugging why your changes don't show.
```
Comment (NataliaLKB) [inline docs/ab-testing.md]: Though a good tip, I don't know if this is the best doc for this. Potentially in the readme is better? We have a section on [troubleshooting](https://github.com/guardian/frontend#troubleshooting)

File: docs/ab-testing.md [hunk 3]
```diff
@@ -136,6 +136,10 @@ can follow our standard [contributing guidelines](https://github.com/guardian/fr
 
 You can stop and start the test using our [switchboard](https://frontend.gutools.co.uk/dev/switchboard).
 
+To see the test in action locally: `localhost:9000/[articleId]#ab-[testName]=[variantName]`
+
+To see the tests you are part of: open the Dev Console -> Resources -> Local Storage -> choose `http://[domain]` (e.g. localhost or www.theguardian.com)  -> check the `gu.ab.participations` row.
```
Comment (rich-nguyen) [inline docs/ab-testing.md]: Can you mention this is in Chrome?

**Comment by rich-nguyen** [comment]
:+1:

**Comment by johnduffell** [comment]
:+1: brilliant, thanks for these updates!

**Pull request merged**